code/modules/hydroponics/grown/tomato.dm

```diff
@@ -137,11 +141,45 @@
 	spawn(30)
 		if(!QDELETED(src))
 			investigate_log("[key_name(user)] released a killer tomato at [COORD(src)]", INVESTIGATE_BOTANY)
-			var/mob/living/simple_animal/hostile/killertomato/K = new /mob/living/simple_animal/hostile/killertomato(get_turf(src.loc))
+			var/mob/living/simple_animal/hostile/killertomato/K = new(get_turf(src.loc))
 			K.maxHealth += round(seed.endurance / 3)
 			K.melee_damage_lower += round(seed.potency / 10)
 			K.melee_damage_upper += round(seed.potency / 10)
 			K.move_to_delay -= round(seed.production / 50)
 			K.health = K.maxHealth
 			K.visible_message("<span class='notice'>The Killer Tomato growls as it suddenly awakens.</span>")
 			qdel(src)
+
+/obj/item/seeds/tomato/killer/hunter
+	name = "pack of hunter-tomato seeds"
+	desc = "These seeds grow into hunter-tomato plants, a \"safer\" breed of killer tomatoes, that primarily seeks out fauna, not humans."
+	icon_state = "seed-killertomato"
+	species = "huntertomato"
+	plantname = "Hunter-Tomato Plants"
+	product = /obj/item/reagent_containers/food/snacks/grown/tomato/killer/hunter
+	mutatelist = list(/obj/item/seeds/tomato/killer)
+
+/obj/item/reagent_containers/food/snacks/grown/tomato/killer/hunter
+	seed = /obj/item/seeds/tomato/killer/hunter
+	name = "hunter-tomato"
+	desc = "I say to-mah-to, you say tom-mae-to... OH GOD IT'S EATING IAN!!"
+
+/obj/item/reagent_containers/food/snacks/grown/tomato/killer/hunter/attack_self(mob/user)
+	if(awakening || isspaceturf(user.loc))
+		return
+	to_chat(user, "<span class='notice'>You begin to awaken the Hunter Tomato...</span>")
+	awakening = 1
+
+	spawn(30)
```

Comment:
Copypasta, you should use addtimer and callback instead.

Reply:
That's fair, I just copy-pasted the code from the regular killer tomato.

Reply:
Done.